libcxx/docs/ReleaseNotes/20.rst

```diff
@@ -38,8 +38,9 @@ What's New in Libc++ 20.0.0?
 Implemented Papers
 ------------------
 
-- P2747R2: ``constexpr`` placement new (`Github <https://github.com/llvm/llvm-project/issues/105427>`__)
+- P0619R4: Reviewing Deprecated Facilities of C++17 for C++20 (`Github <https://github.com/llvm/llvm-project/issues/99985>`__)
 - P2609R3: Relaxing Ranges Just A Smidge (`Github <https://github.com/llvm/llvm-project/issues/105253>`__)
+- P2747R2: ``constexpr`` placement new (`Github <https://github.com/llvm/llvm-project/issues/105427>`__)
```

Comment:
Unrelated changes?

Reply:
Oh, I intentionally sorted the bullets and the sorting itself was perhaps unrelated. Perhaps I should drop the sorting from this PR.

Should we sort the bullets in the release notes at some time?